### ISSUE TITLE
Can you provide the distilled models?

### ISSUE THREAD

**Issue by liguopeng0923** [Hao840/OFAKD#2]
Hi @Hao840 ,

Can you provide the distilled models?  These are important for reproducing your paper.

**Comment by Hao840** [comment]
hi, please give me some details about the experiment that you are reproducing.

**Comment by liguopeng0923** [comment]
Thanks for your reply.
1. I reproduced the experiment of deit-tiny as teacher and resnet18 as student. But it needs more than 100 hours on 4 V100 GPUs, which hinders direct reproduction.
2. I try to give resnet18 a pretrained weight, but the loss is nan in some epochs.
3. Totally, it is difficult to reproduce your paper. I hope you can provide some checkpoints that have been distilled for my evaluation.

**Comment by Hao840** [comment]
hi @liguopeng0923,

I have uploaded the trained resnet18 student model and corresponding log [here](https://github.com/Hao840/OFAKD/releases), where a DeiT-T model is adopted as the teacher model.

(Sorry for that I fail in retrieving the original checkpoint achieved the reported performance in the paper, which is 71.34%. So I can only provide the one achieved a top-1 accuracy of 71.26% for evaluation.)

**Comment by liguopeng0923** [comment]
Thanks very much! 
But will you be releasing other distilled models?  and the other trained models from scratch? They are also important for fair comparisons.

**Comment by Hao840** [comment]
Disclosure of assets from company computers (where all the experimental results are stored) requires a complex process, so I'm sorry we have no plan to release more trained models currently.
